From 4e9f5bc0ff248d90e02a42bc54be8423fb4de33a Mon Sep 17 00:00:00 2001
From: Chad Joan <chadjoan@gmail.com>
Date: Sat, 31 Dec 2016 20:34:22 -0500
Subject: [PATCH] Fixed compile break on musl libc machines.

The compile breakage is usually seen as a "NULL undefined" error
in libmailutils/sockaddr/str.c

diff --git a/libmailutils/sockaddr/str.c b/libmailutils/sockaddr/str.c
index e5bd5a1..7507f23 100644
--- a/libmailutils/sockaddr/str.c
+++ b/libmailutils/sockaddr/str.c
@@ -25,6 +25,7 @@
 #include <netinet/in.h>
 #include <arpa/inet.h>
 #include <netdb.h>
+#include <stddef.h>
 
 #include <mailutils/sockaddr.h>
 #include <mailutils/errno.h>
-- 
2.7.3

